lib/kino/control.ex

```diff
@@ -111,6 +111,114 @@ defmodule Kino.Control do
     new(%{type: :keyboard, events: events})
   end
 
+  @doc """
+  Creates a new form.
+
+  A form is composed of regular inputs from the `Kino.Input` module,
+  however in a form input values are not synchronized between users.
+  Consequently, the form is another control for producing user-specific
+  events.
+
+  Either `:submit` or `:report_change` must be specified.
+
+  ## Options
+
+    * `:submit` - specifies the label to use for the submit button
+      and enables submit events
+
+    * `:report_change` - whether to send new form value whenever any
```

Comment:
Maybe `report_changes`? Also, what do you think about making the same as `reset_on_submit` and making it a list of all fields? Sometimes you may be interested only in certain changes? 🤔

Reply:
Neat! In that case should we send all fields, listed fields or only the changed fields?

Reply:
Gah, this is tricky. I can all cases being useful. For example, for a chat app, we want to know if the message is being typed without necessarily sending it. So in this case, we want to track but not send it!

We could do `[field: true | :only_send | :only_track | false]`, where true stands for send and track, false is none, and we have two intermediate states. I don't think we need to implement this right now... but I want to make sure we avoid changes on the over the wire format. So maybe we keep only true/false in the public API but internally we send it as `field: true` for each field?

Reply:
"Being typed" is tricky since we have debounce. How is `field: true` different from `field: :only_send`, I think both mean we should report the changed value?

Reply:
Ah, I think I got it, send would attach to any change events, while track would actually trigger the events.

Reply:
I made the over the wire format to include a map, since it probably makes more sense, currently only specifying true values.